### DISCUSSION TITLE
Its a question regarding Incubator Mailer

### DISCUSSION THREAD

**Discussion by borisdelev** [phalcon/cphalcon#16335] [General]
Ive checked https://github.com/phalcon/incubator-mailer/tree/v2.x and seems we switch to SwiftMailer. Ive never use it, but why not :)... so, i found the project and see message "Swiftmailer is not maintained anymore. [Use Symfony Mailer](https://symfony.com/doc/current/mailer.html) instead. Read more on [Symfony's blog](http://symfony.com/blog/the-end-of-swiftmailer)." Amm... "Swift Mailer will stop being maintained at the end of November 2021."

My question is... if it is not maintained, why we make that change? 

Thanks :)

**Comment by niden** [comment]
I don't recall when we switched to Switfmailer for this component, I thought that we always used that. We do however need to change it to the Symfony mailer exactly for the reason you mentioned.

**Comment by niden** [comment]
Actually I take that back. The `master` branch use PHPMailer as the mailing component 

https://github.com/phalcon/incubator-mailer/blob/master/composer.json#L35

**Reply by borisdelev** [reply]
Well, okay. Just found a few questions about this. I will give them a link to that thread.

Anyway - so... The main idea is to moving to Symfony Mailer 😊

Great and thank you 👍
All best wishes 😊
